pkgs/applications/science/biology/febio-studio/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchFromGitHub, cmake, gcc, qt515, zlib, libGL, libGLU, openssl, libssh, xorg, wrapQtAppsHook, qtbase}:
+
+stdenv.mkDerivation rec {
+  pname = "FEBioStudio";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "febiosoftware";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256:0r6pg49i0q9idp7pjymj7mlxd63qjvmfvg0l7fmx87y1yd2hfw4h";
+  };
+  patches = [
+    ./febio-studio-cmake.patch
+  ];
+
+  cmakeFlags = [
+    "-DQt_Root=${qt515.full}"
+    "-DQt_Ver=5"
+    "-DNOT_FIRST=On"
+    "-DUSE_SSH=On"
+    "-DCAD_FEATURES=Off"
+    "-DMODEL_REPO=Off"
+    "-DUSE_FFMPEG=Off"
```

Comment:
why are all these things turned off? does a regular user of this program not need them?